hack/quickstart/quickstart-gce.md

```diff
@@ -8,6 +8,28 @@ This can be changed to identify separate clusters.
 export CLUSTER_PREFIX=quickstart
 ```
 
+### Add your SSH public key to project under user core
+
+Quickstart scripts assume you can ssh into the `core` user. Either follow the commands below or go to the web console to add a key for the user `core`.
+
+**WARNING** The following process will clobber existing keys in your project's ssh metadata.
```

Comment:
The option (which defaults to "core" anyway) doesn't change default behavior -- so that seems reasonable. Telling people to stomp on their project wide keys doesn't seem ideal (i'm not really familiar with gce, so I don't know if there are ways to scope this down to per-instance launches or something.